crates/bitwarden-core/src/platform/generate_fingerprint.rs

```diff
@@ -32,17 +44,30 @@ pub(crate) fn generate_fingerprint(input: &FingerprintRequest) -> Result<Fingerp
     })
 }
 
+/// Errors that can occur when computing a fingerprint.
+#[derive(Debug, Error)]
+pub enum UserFingerprintError {
+    #[error(transparent)]
+    Fingerprint(#[from] bitwarden_crypto::FingerprintError),
+    #[error(transparent)]
+    Crypto(#[from] bitwarden_crypto::CryptoError),
```

Comment:
FingerprintError already contains CryptoError, do we need the Crypto variant here?

Reply:
Depends, is it accurate to hide it behind another error just because they are similar types? We could perhaps undo my inversion of FingerPrintError::Crypto and keep Crypto::Fingerprint.